### PR TITLE
Added the warning if there is an error in the initialization of the Twillio ClientApi and fixed other errors in the project.

### PR DESCRIPTION
**Pull request by EdZava**
I throw some contributions to the project so that this project can be used without errors or if there are errors that the developer can know about. 

Like the error when initializing the api client. #118 

I indicate them below each commit:

- chore: Add vscode debug launch
- fix: Warning lints
- chore: Add alert on client api fail on init
- fix: Error React, multiple errors.
- fix: Error Prop Id ToasterPush expect string
- fix: Error React, removed unnecessary "UseEffect"

Thank you so much!

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Comment by berkus** [comment]
Thanks @EdZava we will review it as soon as possible!

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (mmubasher) [inline .vscode/launch.json:1]: Can we get rid of this .vscode directory?

Comment (EdZava) [reply .vscode/launch.json:1]: If you remove this directory, you will not have the file to start the project in debug mode.

How are people going to debug the project?
Will each person who is going to use it have to create it over and over again?

**Comment by EdZava** [comment]
@mmubasher 
What problem can you reproduce?
You can attach more detail or a screenshot to help you.

**Comment by EdZava** [comment]
@mmubasher, @berkus 
Very little attention is paid to contributions to your opensource projects.

**Comment by berkus** [comment]
@EdZava hi, can you please update the PR?

**Comment by EdZava** [comment]
@berkus 
I can not.
You can do the merge yourself. This arrangement was already uploaded in this MR as well.

**Comment by berkus** [comment]
@EdZava why not? You need to resolve conflicts before merging.

**Comment by EdZava** [comment]
@berkus 
no, thanks!

File: .vscode/launch.json (hunk 1 above)
Comment (mmubasher) [reply .vscode/launch.json:1]: Totally code owner's preference, from `chore: Add vscode debug launch` I assumed it was debug code going into main. 
I did not see this committed previously and was of view it got committed by mistake.

**Comment by berkus** [comment]
Since author declined to resolve the conflicts, I am going to close this.

**Pull request closed without merging**